chromium_src/chrome/browser/ui/views/side_search/side_search_browser_controller.cc

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Need to include this header here otherwise SidePanel macro gets undef'ed
```

Comment:
Typically we include the corresponding header first to prevent the redefinitions. Should
#include "chrome/browser/ui/views/side_search/side_search_browser_controller.h" here?

Reply:
thanks!

Reply:
Seems like we also need to keep the browser_view.h include as that also has a macro called SidePanel